packages/space-opera/package.json

```diff
@@ -36,13 +36,13 @@
   "devDependencies": {
     "@google/model-viewer": "^1.7.2",
     "@google/model-viewer-shared-assets": "^0.0.1",
-    "@material/mwc-button": "^0.21.0",
-    "@material/mwc-checkbox": "^0.21.0",
-    "@material/mwc-icon-button": "^0.21.0",
-    "@material/mwc-switch": "^0.21.0",
-    "@material/mwc-tab": "^0.21.0",
-    "@material/mwc-tab-bar": "^0.21.0",
-    "@material/mwc-textfield": "^0.21.0",
+    "@material/mwc-button": "^0.18.0",
+    "@material/mwc-checkbox": "^0.18.0",
+    "@material/mwc-icon-button": "^0.18.0",
+    "@material/mwc-switch": "^0.18.0",
+    "@material/mwc-tab": "^0.18.0",
+    "@material/mwc-tab-bar": "^0.18.0",
+    "@material/mwc-textfield": "^0.18.0",
```

Comment:
I had to downgrade mwc to get our tabs to stop scrolling. There should be a CSS way to fix it in the new version, but I couldn't figure it out.